uploader/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -ex
+curl --header "Content-Type: application/json" \
+      --request POST \
+      --data '{"metadata":{"title":"Title here","upload_type":"poster","description":"This is just for test","creators":[{"name":"Arshul","affiliation":"CROSS"}]}}' \
```

Comment:
These metadata attributes can be exposed as environment variables in the same way that the `ZENODO_API_TOKEN` is exposed as one.